### ISSUE TITLE
*** bit out of range 0 - FD_SETSIZE on fd_set ***: terminated Running the AppImage

### ISSUE THREAD

**Issue by victorguidi** [spacedriveapp/spacedrive#1654]
### Describe the bug

Hello everyone, 

When running the AppImage on my machine I got the following error: 

2023-10-21T20:54:39.310948Z  INFO sd_core: Starting core with data directory '/home/kun/.local/share/spacedrive'
2023-10-21T20:54:39.345935Z  INFO sd_core::p2p::p2p_manager: Node '12D3KooWLYNFppQUoKPcJFCny3ywaqxr8uWgkTTiXQCRUaFzDaWp' is now online listening at addresses:
 {}
2023-10-21T20:54:39.523083Z  INFO sd_core: Spacedrive online.
2023-10-21T20:54:39.525977Z  INFO sd_desktop::tauri_plugins: Internal server listening on: http://127.0.0.1:40073
*** bit out of range 0 - FD_SETSIZE on fd_set ***: terminated
[1]    10725 IOT instruction (core dumped)  ./Downloads/Spacedrive-linux-x86_64.AppImage

I was about to try for the first time, not sure if that's an issue on my machine or if is a known issue for the application.

Since the FD_SETSIZE is a know limitation on Linux machines I am not sure if might be happening only on my machine. 

This is my kernel information: 

Linux version 6.1.58-1-lts (linux-lts@archlinux) (gcc (GCC) 13.2.1 20230801, GN
U ld (GNU Binutils) 2.41.0) #1 SMP PREEMPT_DYNAMIC Sun, 15 Oct 2023 18:38:43 +0
000

### Reproduction

Download the appimage

chmod +x "downloaded appimage"

./appimage

### Expected behavior

Print the message: 

*** bit out of range 0 - FD_SETSIZE on fd_set ***: terminated

### Platform and versions

```Shell
8.8.0
cargo 1.72.1 (103a7ff2e 2023-08-15)
rustc 1.72.1 (d5c2e9c34 2023-09-13)
```

**Comment by eboye** [comment]
Same here

```
2023-10-22T02:03:01.145711Z  INFO sd_core: Starting core with data directory '/home/[username]/.local/share/spacedrive'
2023-10-22T02:03:01.187853Z  INFO sd_core::p2p::p2p_manager: Node '12D3KooWRmbgwuD63em9HARnMoRSo26Z2wx772F6Xsa3SpS3HhWr' is now online listening at addresses: {}
2023-10-22T02:03:01.647696Z  INFO sd_core: Spacedrive online.
2023-10-22T02:03:01.652834Z  INFO sd_desktop::tauri_plugins: Internal server listening on: http://127.0.0.1:34153
*** bit out of range 0 - FD_SETSIZE on fd_set ***: terminated
[1]    21380 IOT instruction (core dumped)  ./gearlever_spacedrive_7aeb5c.appimage
```
uname -a

``Linux stupid-dragon 6.5.8-arch1-1 #1 SMP PREEMPT_DYNAMIC Thu, 19 Oct 2023 22:52:14 +0000 x86_64 GNU/Linux``

Arch Linux with Gnome 45

**Comment by HeavenVolkoff** [comment]
Thank you for reporting this issue. I can confirm this is happening on the latest release version of AppImage. I will look into it

**Comment by HRKings** [comment]
Hey! I don't know if it is related, but I get the same error when using the AppImage, but when installing the package directly I only get a Segmentation Fault and nothing more. Should I open another bug? And there is something that I can do to gather any more info ?

**Comment by HeavenVolkoff** [comment]
Hi @HRKings, both the `deb` and the `Appimage` are broken rn. I am working to fix the `deb` (which is used to build the `Appimage`). Currently, I am finishing #1685 to fix the FFMpeg linking issues present in the `deb`, then I need to fix some wrong file permission while packaging the `deb` (https://github.com/spacedriveapp/spacedrive/issues/1512#issuecomment-1758550164), then I will start working on a fix for the `AppImage`, my current plan is to move our build to use [appimage-builder](https://appimage-builder.readthedocs.io/en/latest/examples/gnome.html), instead of the LinuxDeploy script that is currently used (which seems to be the culprit for some of the problems we are facing). I am sorry that we don't have a working updated build of Spacedrive for Linux available at the moment, but I  hope to get them back into working condition soon

**Comment by brxken128** [comment]
This will be fixed in #1785.
